### DISCUSSION TITLE
Code exploration tools

### DISCUSSION THREAD

**Discussion by kcramakrishna** [Digital-Assistant/Digital_Assistant_Client#296] [General]
https://www.redotheweb.com/CodeFlower/
https://www.redotheweb.com/DependencyWheel/

https://www.sourcetrail.com/
https://www.scitools.com/

This seems great:
https://about.sourcegraph.com/
https://github.com/sourcegraph/sourcegraph

GNU listing of tools:
https://www.gnu.org/software/global/links.html

**Comment by yureshwar** [comment]
For javascript UML diagrams

https://www.npmjs.com/package/arkit
